_includes/components/image-example-correct.html

```diff
@@ -0,0 +1,24 @@
+{% comment %}
+include
+- image
+- text
+{% endcomment %}
+<div class="usa-image-example--correct">
+    <figure class="usa-image-example__figure">
+      <img class="usa-image-example__image" alt="correct example, use dark background" src="{{ include.image | prepend: site.baseurl}}" />
```

Comment:
Acknowledging this is still draft, but flagging that we'll want to translate all the English text in this file.